File: arkts_static_test/03_Types/ets_cases/3.17.2_Type_Tuple/cross_lang_verify/runtime/TYP_03_17_02_008.swift
/**
 * Swift cross-language verification for TYP_03_17_02_008_RUNTIME_EMPTY_TUPLE
 */
import Foundation

let empty: () = ()

// Swift 空元组是 Any 类型
if !(empty is Any) {
    fatalError("empty should be Any")
}

print("verified")

File: arkts_static_test/03_Types/ets_cases/3.17.2_Type_Tuple/cross_lang_verify/runtime/TYP_03_17_02_008.java
/**
 * Java cross-language verification for TYP_03_17_02_008_RUNTIME_EMPTY_TUPLE
 */
class TYP_03_17_02_008 {
    public static void main(String[] args) {
        Object[] empty = {};

        if (!(empty instanceof Object)) {
            throw new AssertionError("empty should be Object");
        }

        System.out.println("verified");
    }
}
